src/scripts/sync_teams/sync_teams.py

```diff
@@ -174,11 +182,33 @@ def process_teams(token, organization, assets: List[Asset]) -> List[str]:
     teams_to_create = get_teams_to_create(desired_teams, existing_team_names)
     teams_to_delete = get_teams_to_delete(desired_teams, existing_team_names)
     if teams_to_create:
-        logger.info(f"Creating {len(teams_to_create)} team(s): {teams_to_create}")
+        logger.info(
+            f"Creating {len(teams_to_create)} team(s): {teams_to_create}")
         create_teams(token, teams_to_create)
     return teams_to_delete
 
 
+def process_members(token: str, organization: Organization, existing_teams: List[TeamAttributes]) -> None:
+    logger.info("Processing team members.")
+    for team_structure in organization.teams:
```

Comment:
shouldn't we use the get_desired_teams logic here (same as when processing teams)?

Reply:
No, since this is the place where we go over the members of each team in the JSON. The `existing_teams` parameter is the one that contains the teams from the DB.

Reply:
No, since this is the place where we go over the members of each team in the JSON. The existing_teams parameter is the one that contains the teams from the DB.

Reply:
You're right, while I do want to go over the organization.teams to get the members, I do also need to pass the desired teams and check if the team is there before working on it. Fixed.